src/test/ui/macros/best-failure.stderr

```diff
@@ -0,0 +1,21 @@
+error: no rules expected the token `$`
+  --> $DIR/best-failure.rs:4:30
+   |
+LL | macro_rules! number {
+   | ------------------- when calling this macro
+...
+LL |         number!(neg $signed, $self);
```

Comment:
kind of a shame that we highlight the whole `$self` metavariable rather than just the `$` token given the primary message of this diagnostic, but w/e